src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -840,6 +843,8 @@ maybe_convert_reductions({KVs0, UserReductions}) ->
 
 maybe_convert_kv({<<"_local/", _/binary>> = DocId, _}) ->
```

Comment:
Shouldn't we convert this clause to `maybe_convert_kv(#doc{id = <<"_local/", _/binary>>} = Doc) ->`?